common/production_methods/06_urban_center.txt

```diff
@@ -713,7 +713,7 @@ pm_dry_steam_stations = {
 pm_binary_cycle_plants = {
```

Comment:
```suggestion
pm_enhanced_geothermal_systems = {
```

Reply:
This is my mistake, as `binary_cycle_plants` was renamed to this tech: `enhanced_geothermal_systems`